web/blackjack/src/app/game/game/game.component.ts

```diff
@@ -42,11 +42,21 @@ export class GameComponent implements OnInit {
       this.fetchGame();
     });
 
-    interval(500).subscribe(() => {
+    this.intervalSubscription = interval(500).subscribe(() => {
       this.fetchGame();
     });
   }
 
+  ngOnDestroy(): void {
+    this.intervalSubscription?.unsubscribe();
+  }
+
+  @HostListener('window:beforeunload', [ '$event' ])
```

Comment:
When I run the game and try to either go back home or close the tab, there is no HTTP `DELETE` function on the network, nor does my console function print. Should I be using the `unload` event, or am I not implementing this correctly?